lib/Walli.ts

```diff
@@ -12,10 +13,13 @@ class Walli {
 
   private logger: Logger;
   private btcdClient: BtcdClient;
+  private config: Config;
 
-  constructor(config: Config) {
-    this.logger = new Logger(config.logfile, config.loglevel);
-    this.btcdClient = new BtcdClient(config.rpc);
+  constructor(config: ConfigType) {
+    this.config = new Config().load(config);
+    console.log(this.config);
```

Comment:
Please remove this `console.log` statement in the next commit. And what code editor are you using? If it is Visual Studio Code I could commit my configurations for debugging.

Reply:
Oops didn't notice that and use I'm using VS Code.